Automattic/pocket-casts-ios
# Folders: limit folder's name to 100 chars

Fixes #69 

Add a limit of 100 characters for the folder's name.

In order to make testing easier, you can change `maximumAllowedCharactersForName` to a lower number (like `5`).

Example using 5:

https://user-images.githubusercontent.com/7040243/179272540-8460ca1c-6194-41a9-aa73-ce2f899cb4c6.mp4

## To test

1. Run the app
2. ✅ Create a new folder and make sure the max limit is 100 (or the number you defined in `maximumAllowedCharactersForName`)
3. Edit the folder
4. ✅ Make sure the max char limit respects `maximumAllowedCharactersForName`

Given it's a SwiftUI/Combine change, please test on iOS 14 **and** iOS 15.
